Otel doc fixes (#10150)
opensearch-project/documentation-website: _data-prepper/pipelines/configuration/sources/otel-logs-source.md

| unframed_requests | Boolean | Enables requests that are not framed using the gRPC wire protocol. Default value is `false`. |
| thread_count  | int | The number of threads to keep in the `ScheduledThreadPool`. Default value is `500`. |
| max_connection_count | int | The maximum number of open connections allowed. Default value is `500`. |

> | output_format | String | Indicates the output format of the events to be generated. Allowed values are `otel` or `opensearch`. Default value is `opensearch`. |

Reviewer: ```suggestion
| `output_format` | String | Specifies the output format of the generated events. Valid values are `otel` or `opensearch`. Default is `opensearch`. |

```

    - otel_logs_source:
```

> Use the `otel` output format option as below if open telemetry standard format is desired.

Reviewer: ```suggestion
"To generate data in OpenTelemetry format, use the `otel` value for the `output_format` as shown in the following example:"
```
